virttest/libvirt_vm.py

```diff
@@ -2033,6 +2033,16 @@ def migrate(self, dest_uri="", option="--live --timeout 60", extra="",
         # Since dest_uri could be None, checking it is necessary.
         if result.exit_status == 0 and dest_uri:
             self.connect_uri = dest_uri
+
+        # Set vm name in case --dname is specified.
+        migrate_options = ""
+        if option:
+            migrate_options = str(option)
+        if extra:
+            migrate_options += " %s" % str(extra)
```

Comment:
`str` is not necessary in this line.

Reply:
`%s` does the typecast, so we don't have to do twice.

Reply:
Updated. Thanks.